packages/lesswrong/server/apolloServer.ts

```diff
@@ -340,6 +340,9 @@ export function startWebserver() {
   app.get('*', async (request, response) => {
     response.setHeader("Content-Type", "text/html; charset=utf-8"); // allows compression
 
+    if (!getPublicSettingsLoaded()) throw Error('Failed to render page because publicSettings have not yet been initialized on the server')
+    const publicSettingsHeader = `<script> var publicSettings = ${JSON.stringify(getPublicSettings())}</script>`
```

Comment:
This should probably use `embedAsGlobalVar` (I know this code just moved from a different file, but I think the point still stands).